Services/TimeManagementService/Validators/EditWorkTimeRequestValidator.cs

```diff
@@ -0,0 +1,46 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.TimeManagementService.Models;
+using System;
+
+namespace LT.DigitalOffice.TimeManagementService.Validators
+{
+    public class EditWorkTimeRequestValidator : AbstractValidator<EditWorkTimeRequest>
+    {
+        public EditWorkTimeRequestValidator()
+        {
+            RuleFor(wt => wt.Id)
+                .NotEmpty()
+                .WithMessage("WorkTimeId must not be empty");
+
+            RuleFor(wt => wt.WorkerUserId)
+                .NotEmpty()
+                .WithMessage("WorkerUserId must not be empty");
+
+            RuleFor(wt => wt.StartTime)
+                .NotEqual(new DateTime())
+                .WithMessage("StartTime must not be empty");
+
+            RuleFor(wt => wt.EndTime)
+                .NotEqual(new DateTime())
+                .WithMessage("EndTime must not be empty");
```

Comment:
More check. See AddWorkTimeRequestValidator and it would be cool to apply this: https://docs.fluentvalidation.net/en/latest/rulesets.html

Reply:
I think, we already have enough rules

Reply:
If you change working hours, then you can break some of the rules. For example:
![image](https://user-images.githubusercontent.com/50001467/91963492-98fcf700-ed16-11ea-8b95-f5313e77e402.png)
![image](https://user-images.githubusercontent.com/50001467/91963549-a74b1300-ed16-11ea-98f0-e92aef9ed0fc.png)
![image](https://user-images.githubusercontent.com/50001467/91963641-ca75c280-ed16-11ea-9576-7c4758900b9a.png)

Reply:
And if your new checks do not differ from the old ones, then you can use the method presented in the link.

Reply:
added some rules

Reply:
There is not enough intersection rule

Reply:
okey we will talk about it in call

Reply:
fixed